enterprise/app/code/code.tsx

```diff
@@ -135,14 +136,68 @@ export default class CodeComponent extends React.Component<Props, State> {
     let bytestreamURL = this.props.search.get("bytestream_url");
     let invocationID = this.props.search.get("invocation_id");
     let filename = this.props.search.get("filename");
-    if (this.isSingleFile()) {
+    if (this.isSingleFile() && bytestreamURL) {
       rpcService.fetchBytestreamFile(bytestreamURL, invocationID, "text").then((result: any) => {
         this.editor.setModel(monaco.editor.createModel(result, undefined, monaco.Uri.file(filename || "file")));
       });
       return;
     }
 
+    let lcovURL = this.props.search.get("lcov");
+    let commit = this.props.search.get("commit");
+    if (this.isSingleFile() && lcovURL) {
+      this.octokit.rest.repos
+        .getContent({
+          owner: this.currentOwner(),
+          repo: this.currentRepo(),
+          path: this.currentPath(),
+          ref: commit || this.state.commitSHA || "master",
```

Comment:
Am I misreading or does this get messed up if the user has added / removed lines from the file?

Reply:
Sorry, I meant to say, if the user has added / removed lines *and not pushed to github yet*

Reply:
That's right, if it's not a CI run - but a local run and the changes haven't been published to Github, the source file might be out of date. I'm not sure there's much we can do in that scenario unless Bazel sends us the source file (which it likely does in the remote execution case, so that's an optimization we could make later).

Reply:
I don't really care about people who are hacking the URL up, but you could check that the revision actually exists on the coverage card and not link out to the code view if the file wasn't found?

Reply:
I don't really want to do any requests to Github from outside of the code UI. 

You cool with seeing if this becomes a common issue, and if it is we can come up with something fancier?

Reply:
Slash, will you let me get this into tomorrow's release and I promise to make improvements in the future :)